sklearn/impute.py

```diff
@@ -80,10 +82,10 @@ class SimpleImputer(BaseEstimator, TransformerMixin):
 
     Parameters
     ----------
-    missing_values : integer or "NaN", optional (default="NaN")
+    missing_values : real number, string, np.nan or None,
+        optional (default=np.nan).
```

Comment:
Sphinx does not really like this way of wrapping the line .. 
If you want it to render nicely we can do

```
    missing_values : real number, string, np.nan or None, \
optional (default=np.nan).
         The placeholder for ...
```